docs/docs/index.md

```diff
@@ -4,13 +4,6 @@ order: 1
 sidebar_position: 1
 ---
 
-:::tip
```

Comment:
@MSalopek is there something else that needs to be done for the docs?

Reply:
Cosmwasm and build section updates. Those can be main only